### ISSUE TITLE
Open-Source Coronavirus Monthly Update Meeting #5: 17:00-18:00 BST, Monday 10th October 2022

### ISSUE THREAD

**Issue by TomkUCL** [StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B#29]
If you like SARS-CoV-2 proteins, drug discovery, and open science, then join the SGC online, and the sgc-ocn team meeting today from 17:00-18:00 BST to discuss pan-coronavirus antivirals. 

Join Zoom Meeting: https://ucl.zoom.us/j/97172937586

**Comment by mattodd** [comment]
OK, can you post an agenda (just text in one of these boxes - easier to search for (using e.g. Google) than a PDF (which Google can't see)). I'd like to include:

1) The recent inhibitor you found @TomkUCL  https://github.com/StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B/issues/22#issuecomment-1270191492
2) Where we're up to with compounds from Enamine etc @ahsgc 
3) Update on your chem, @TomkUCL 
4) Update on SPR (@ClaratheChemist feel free to come along!)
5) Update on Review - @TomkUCL could you post links to any and all active documents please?

Anything from previous meetings in Aug and Sept we need to resolve?

Pinging the team, which you can do like this: @StructuralGenomicsConsortium/cnp4-nsp13

**Comment by TomkUCL** [comment]
Today's monthly meeting agenda will include the following:

**Setting up the Nsp13 SPR assay (17:00-17:10)** @TomkUCL @ClaratheChemist

- S Series SA chips ordered - waiting for delivery. 
- Nsp13 protein is now at UCL being stored at -70degC.
- 51 compounds from Enamine in DMSO are now at UCL, stored at -20 degC. 
- Status of reference compounds; ADP in-house, AMP-PNP due for delivery. 
- Joe Newman's fragments - do we need them / can we get a hold of them? 
- Discussion of Nsp13 SPR approach published Oct 2022 and ordering of control compound (PUG); https://github.com/StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B/issues/22#issuecomment-1270191492. 
- Discussion of Biotin capture method and blank channel runs. 
- See here for further updates: SPR Assay - Getting up and running #25

**Updates on delivery of Enamine Compounds (17:10-17:20** @ahsgc 

**Updates on the synthesis of generative compounds (17:20-17:30)** @TomkUCL 

- Further work is needed on reductive amination chemistry of the penultimate step for generative compound no. 1, NaBH(OAc)3 is too mild, screen NaBH(CN)3 and NaBH4 this week. 
- Scale up of a key intermediate of generative compounds list: TK-41-3: Preparation of ethyl 2-(5-fluoropyridin-2-yl)acetate. https://uk-mynotebook.labarchives.com/share/Thomas%2520Knight/MTI0LjgwMDAwMDAwMDAwMDAxfDEwNDQxLzk2L1RyZWVOb2RlLzM1ODI4MzU2Mzl8MzE2Ljc5OTk5OTk5OTk5OTk1 (5-10 g needed). Any updates from CRO?

**Comment by TomkUCL** [comment]
Slides from today's meeting: [Monthly update 10.10.2022.pdf](https://github.com/StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B/files/9747725/Monthly.update.10.10.2022.pdf)

Meeting recording:

Meeting minutes:

**Comment by mattodd** [comment]
Action item on one of us to catch up with Joe Newman about the value of ADP as a control, given its relatively high off rate. Though I think we've some interesting choices here that should allow us to validate the SPR assay.

Action on you @TomkUCL to get together the two sheets of compounds that we sent to Enamine - one commercial, one generative - and delineate which have been sent, which will not be coming (with help from @ahsgc) and which we're still waiting for. Also highlight in each sheet which compounds still need synthesis (either us or Pyramal). 

@tmw20653 will catch up with Matthieu and team about the use of Helihub to direct us to the most relevant examples if inhibitors for the review.

**Comment by jamesday100** [comment]
Hi All, sounded like a productive meeting with a good discussion. And happy to hear that there could be some potential tool molecules. I am still in the process of reading the papers. Did we find fragments in the same potential binding site as the PUG molecule?

@TomkUCL: PS: For tricky reductive aminations, often you can use titanium isopropoxide to force the imine to form.

**Comment by jamesday100** [comment]
Thanks for the additional information and beautiful images. 

PS for the titanium isopropoxide reductive amination, I would prefer the free base for these reactions. Often the TFA salt will interfere in the reaction.

**Comment by TomkUCL** [comment]
No problem - Chimera X is great for getting pretty images like this from PDB structures. 

I'm having some trouble with this currently. I'm trying to convert the TFA salt of the amine below to the free base but as you can imagine it is very hydrophilic, so a typical base wash with hydroxide or carbonate takes all of the amine into the aqueous phase. Do you know a potential solution to get around this? I've considered using a dowex resin, but wondered if this is the easiest thing to try based on your experience with small primary amines.. Thanks. 

![image](https://user-images.githubusercontent.com/92033163/196913879-e6337371-5323-4a8f-915f-4dde4bdc408c.png)

**Comment by toluene44** [comment]
One method I’ve used in the past is to liberate the free base in brine, then extract with iso-propanol. Iso-propanol is not soluble in brine and this is a good method for extracting water-soluble compounds

Peter

From: Tom Knight ***@***.***>
Date: Thursday, October 20, 2022 at 6:07 AM
To: StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B ***@***.***>
Cc: Brown, Peter J ***@***.***>, Team mention ***@***.***>
Subject: Re: [StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B] Open-Source Coronavirus Monthly Update Meeting #5: 17:00-18:00 BST, Monday 10th October 2022 (Issue #29)

No problem - Chimera X is great for getting pretty images like this from PDB structures.

I'm having some trouble with this currently. I'm trying to convert the TFA salt of the amine below to the free base but as you can imagine it is very hydrophilic, so a typical base wash with hydroxide or carbonate takes all of the amine into the aqueous phase. Do you know a potential solution to get around this? I've considered using a dowex resin, but wondered if this is the easiest thing to try based on your experience with small primary amines.. Thanks.

[image]<https://user-images.githubusercontent.com/92033163/196913879-e6337371-5323-4a8f-915f-4dde4bdc408c.png>

**Comment by jamesday100** [comment]
> No problem - Chimera X is great for getting pretty images like this from PDB structures.
> 
> I'm having some trouble with this currently. I'm trying to convert the TFA salt of the amine below to the free base but as you can imagine it is very hydrophilic, so a typical base wash with hydroxide or carbonate takes all of the amine into the aqueous phase. Do you know a potential solution to get around this? I've considered using a dowex resin, but wondered if this is the easiest thing to try based on your experience with small primary amines.. Thanks.
> 
> ![image](https://user-images.githubusercontent.com/92033163/196913879-e6337371-5323-4a8f-915f-4dde4bdc408c.png)

Yes I can see why that might be an issue. Often, for polar amines we use SCX columns (Silica-Ph-SO3H) washing through with methanol to trap the amine and release with 2M ammonia in methanol. Or you can trap the acid with (Silica-NH2) and wash with methanol to elute the free base. 

Additionally for polar amines, you can use 10% iPrOH in CHCl3 in saturated brine solution.

**Comment by toluene44** [comment]
I’d just make up 2N NaOH in brine

Peter

From: Tom Knight ***@***.***>
Date: Thursday, October 20, 2022 at 7:07 AM
To: StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B ***@***.***>
Cc: Brown, Peter J ***@***.***>, Mention ***@***.***>
Subject: Re: [StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B] Open-Source Coronavirus Monthly Update Meeting #5: 17:00-18:00 BST, Monday 10th October 2022 (Issue #29)

@toluene44<https://github.com/toluene44> thanks - this step is following from a TFA de-Boc, so presumably I'll need to neutralise the TFA salt with a with base wash before I can do this extraction?

**Comment by TomkUCL** [comment]
Thanks Peter - this seems to have done the trick.
